plugin/adsk/scripts/mayaUsdMenu.mel

```diff
@@ -639,7 +715,8 @@ global proc mayaUsdMenu_duplicateToUSD( string $proxy, string $obj )
     string $objLong[] = `ls -l $obj`;
     string $proxyLong[] = `ls -l $proxy`;
     if (size($objLong) && size($proxyLong)) {
-        mayaUsdDuplicate $objLong[0] $proxyLong[0];
+        // TODO: show options.
```

Comment:
The story is only to cover the merge-to-USD. I left a TODO if we want to do the same to the duplicate command.